examples/gno.land/p/nt/commondao/proposal.gno

```diff
@@ -219,6 +225,8 @@ func (p Proposal) IsVoteChoiceValid(c VoteChoice) bool {
 	return p.voteChoices.Has(string(c))
 }
 
+// add default prop render
```

Comment:
Definitely a good feature to have, default render support for proposals, or DAOs so users can opt in.
It would be good to consider having a `p/nt/commondao/ui` or `p/nt/commondaoui` for render related features and default render logic.

At some point I tried to have render defaults so they could also be used in `commondao` realm, but I found it was somehow limiting when adding links between types live DAO, proposal and vote for example, or when dealing with pagination because the rendered content was highly dependent on the realm render logic and the paths being resolved.